src/main/java/com/blamejared/compat/thermalexpansion/Refinery.java

```diff
@@ -54,23 +64,56 @@ protected String getRecipeInfo() {
             return LogHelper.getStackDescription(output);
         }
     }
-    
+
+    private static class AddPotion extends BaseAction {
+
+        private FluidStack output, input;
```

Comment:
If you never change them, make them final ^^

Reply:
Again, the convention of all of the other thermalexpansion handlers is that they aren't final. I don't feel like it's my place to begin changing that.